index.html

```diff
@@ -1403,23 +1415,36 @@
             Let <dfn>presentation message data</dfn> be the payload data to be
             transmitted between two browsing contexts. Let <dfn>presentation
             message type</dfn> be the type of that data, one of
-            <code>text</code> and <code>binary</code>.
+            <code>text</code> or <code>binary</code>.
           </p>
           <p>
-            When the user agent is to <dfn data-lt="algorithm-send">send a
-            message through a <code>PresentationConnection</code> S</dfn>, it
-            MUST run the following steps:
+            When the user agent is to <dfn data-lt="send-algorithm">send a
+            message</dfn> through a <a>presentation connection</a>, it MUST run
+            the following steps:
           </p>
+          <dl>
+            <dt>
+              Input
+            </dt>
+            <dd>
+              <code>presentationConnection</code>, the <a>presentation
+              connection</a> connected to the other browsing context
+            </dd>
+            <dd>
+              <code>messageOrData</code>, the <a>presentation message data</a>
+              to send to the other browsing context
+            </dd>
+          </dl>
           <ol link-for="PresentationConnection">
-            <li>If the <a>state</a> property of <a>PresentationConnection</a>
-            is not <code>"connected"</code>, throw an
-            <code>InvalidStateError</code> exception.
+            <li>If the <a>state</a> property of
+            <code>presentationConnection</code> is not <code>connected</code>,
```

Comment:
In #214 we speculate about allowing messaging to/from PresentationConnection in a "connecting" state. Do you plan to add this in a follow up PR? Without this difference from "disconnected" state, it's not clear why we just not resolve the start() promise with the "disconnected" state that changes to "connected" later on. All I can think of is it will be clearer for the Web developer whether to expect the transition to "connected" or not but it might not be good enough to justify adding a new state then.